src/main/kotlin/no/nav/syfo/behandlendeenhet/api/BehandlendeEnhetDTO.kt

```diff
@@ -5,3 +5,24 @@ data class BehandlendeEnhetDTO(
     val isNavUtland: Boolean,
     val oppfolgingsenhet: String? = null,
 )
+
+data class TildelOppfolgingsenhetRequestDTO(
+    val personidenter: List<String>,
+    val oppfolgingsenhet: String,
+)
+
+data class TildelOppfolgingsenhetResponseDTO(
+    val tildelinger: List<TildelOppfolgingsenhetDTO>,
+    val errors: List<ErrorDTO>,
+)
+
+data class TildelOppfolgingsenhetDTO(
+    val personident: String,
+    val oppfolgingsenhet: String?,
+)
+
+data class ErrorDTO(
+    val personident: String,
+    val errorMessage: String? = null,
+    val errorCode: Int? = null,
```

Comment:
Nå kan vi godt kjøre på med dette 😊 Er det tenkt at dette skal vises til brukerne? Hvis vi vet backend hva som er grunnen til feilen så er vi nok mest sannsynlig også kjent med dette frontend (utenom tilgang kanskje?), og at vi da heller burde validere frontend. 
Har vært borti før at det lages utbroderende feil-DTO'er som egentlig bare blir brukt av utviklerne. Å vite hvem som feilet kan jo være greit, men at responsen skal inneholde HTTP feilkode i et kall som faktisk returneres som HTTP 200 høres litt merkelig ut 🤔
Som sagt så kan vi for nå godt kjøre på med dette, bare litt usikker på hva vi får av å returnere en liste med feil 😊

Reply:
Så blir det jo litt kompleksitet i api-laget rundt det også 😬